### ISSUE TITLE
Change log support

### ISSUE THREAD

**Issue by joebeijing** [toeverything/AFFiNE#1594]
This is mainly used to showcase what AFFiNE has recently updated.

**Comment by xell** [comment]
Please fix the UI bug:

![CleanShot 2023-05-15 at 12 00 46@2x](https://github.com/toeverything/AFFiNE/assets/132558/7a3b8a7a-0fda-4943-a1d4-03222a64d2d8)

**Comment by pengx17** [comment]
I think in the new design, changelog will be merged into the app updater button so that this one was recently removed from the latest version. cc @hwangdev97

**Comment by JimmFly** [comment]
> I think in the new design, Change Log component will be merged into the app updater button so that this one was recently removed from the latest version. cc @hwangdev97

Does it still need to be displayed on the web?

**Comment by hwangdev97** [comment]
I think both the website and the client should display the new design, the new design should be structured as follows:

<img width="703" alt="shot-2023-05-15-13 17 35@2x" src="https://github.com/toeverything/AFFiNE/assets/24713927/3ff2e011-8029-410d-9957-7cf3bdf848ef">

**Comment by pengx17** [comment]
@m1911star will you be able to make the ui here as well?

**Comment by m1911star** [comment]
> @m1911star will you be able to make the ui here as well?

I'm not entirely sure when the deadline is. Will it be released together with the beta?

**Comment by pengx17** [comment]
> I'm not entirely sure when the deadline is. Will it be released together with the beta? @xell @pengx17

Yeah, I hope so. I think the UI part shall be trivial, but you may need to handle different states.

**Comment by xell** [comment]
> I'm not entirely sure when the deadline is. Will it be released together with the beta? @xell @pengx17 

The deadline of beta is May 18 17:30 UTC+8. Please refer [the internal doc](https://toeverything.feishu.cn/wiki/OyKBw7yJhiXQJVkyogacP7tZnEc) (you can request permission from me).

**Comment by m1911star** [comment]
Sorry. I cant catch up with beta release date these two days. Maybe will find some other bugs to fix later.

**Comment by xell** [comment]
> Sorry. I cant catch up with beta release date these two days. Maybe will find some other bugs to fix later. 

Ok. It's understandable. Thanks for all the work.

**Comment by pengx17** [comment]
No worries @m1911star, I can take over the task
